scm/driver/harness/webhook.go

```diff
@@ -271,6 +275,7 @@ func convertPullReq(pr pullReq, ref ref, commit hookCommit) scm.PullRequest {
 		Closed: pr.State != "open",
```

Comment:
Do we need `pr.State == "closed"`?

Reply:
agree

Reply:
pr merged is also pr closed, that is what we are doing in github's case too.

Reply:
can change it.